zap/zap.go

```diff
@@ -0,0 +1,97 @@
+package zap
+
+import (
+	"errors"
+
+	"github.com/airbrake/gobrake/v5"
+	"go.uber.org/zap/zapcore"
+)
+
+type Option func(*Core)
+
+type Core struct {
+	zapcore.LevelEnabler
+
+	Notifier *gobrake.Notifier
+
+	coreFields map[string]interface{}
+	depth      int
+}
+
+func NewCore(enab zapcore.LevelEnabler, notifier *gobrake.Notifier, options ...Option) (*Core, error) {
```

Comment:
NewCore accepts options. Can you, please, provide an example in `examples/zap` about how to pass these options?

Reply:
I will be removing the `options` as I don't see it's necessity for any extraordinarily configuration.